build-support/bin/generate_travis_yml.py

```diff
@@ -225,13 +225,20 @@ def __str__(self) -> str:
     return str(self.value)
 
 
-def _linux_before_install(include_test_config: bool = True) -> List[str]:
+def _linux_before_install(include_test_config: bool = True, travis_wait: bool = False) -> List[str]:
```

Comment:
I recommend renaming to `install_travis_wait`. At first glance, I thought this meant ~`run travis_wait`

Reply:
good point!